core/common/src/main/java/alluxio/master/MasterInquireClient.java

```diff
@@ -110,6 +110,9 @@ public static ConnectDetails getConnectDetails(AlluxioConfiguration conf) {
       if (conf.getBoolean(PropertyKey.ZOOKEEPER_ENABLED)) {
         return new ZkMasterConnectDetails(conf.get(PropertyKey.ZOOKEEPER_ADDRESS),
             conf.get(PropertyKey.ZOOKEEPER_LEADER_PATH));
+      } else if (ConfigurationUtils.getMasterRpcAddresses(conf).size() > 1) {
```

Comment:
Was this a bug? How did this show up?

Reply:
Yes this was a bug. `getConnectDetails` did not function properly if the master was configured for the embedded journal. I added in the conditional check to return the proper connect details. The logic for determining the proper type is analogous to the `MasterInquireClient#create()` function